services/vod-to-live/service/.env.template

```diff
@@ -0,0 +1,45 @@
+NODE_ENV=development
+SERVICE_ID=vod-to-live-service
+LOG_LEVEL=DEBUG
+
+PORT=11900
+
+# This variable is optional and is only needed for local development
+DEV_CHANNEL_SERVICE_BASE_URL=https://channel.service.eu.axinom.net
+
+# Secret for signing and verifying the webhook messages
+# In development, fill the values using `yarn setup:webhook` script
+PRE_PUBLISHING_WEBHOOK_SECRET=
+
+# Base URL to the Unified Virtual Channel Management API
+VIRTUAL_CHANNEL_MANAGEMENT_API_BASE_URL=
+
+# Optional. Used for authorization by API key with USP Channel Management API
+VIRTUAL_CHANNEL_MANAGEMENT_API_KEY=
+
+# Base URL to the Unified Streaming Origin, or CDN for Origin
+VIRTUAL_CHANNEL_ORIGIN_BASE_URL=
+
+# Azure Storage Configuration
+AZURE_STORAGE_CONNECTION=
+AZURE_BLOB_CONTAINER_NAME=
+
+# Live stream DRM protection
+KEY_SERVICE_API_BASE_URL=
+KEY_SERVICE_TENANT_ID=
+KEY_SERVICE_MANAGEMENT_KEY=
+DRM_KEY_SEED_ID=
+
+# Feature flag. When turned on every playlist with duration under 24 hours
+# will be automatically prolonged to hit 24 hour duration.
+# Defaults to TRUE
+PROLONG_PLAYLIST_TO_24_HOURS=TRUE
```

Comment:
@Zehelein I would either change the default to FALSE, or add a comment to Strongly recommend setting the value to FALSE during development. When it is set to true, Virtual Channel would take an eternity to generate a 24h long livestream from the playlist, so you pretty much have to set it to false to be able to test.

Reply:
Ok yes - likely best to default it to false. During development "nobody" will fill 24 hours and for production they should anyways try to manually fill 24 hours (for a 24/7 live stream).